packages/std/src/imports.rs

```diff
@@ -75,6 +111,43 @@ impl Storage for ExternalStorage {
             write_db(key_ptr, value_ptr);
         }
     }
+
+    fn remove(&mut self, key: &[u8]) {
+        // keep the boxes in scope, so we free it at the end (don't cast to pointers same line as build_region)
+        let key = build_region(key);
+        let key_ptr = &*key as *const Region as *const c_void;
+        unsafe {
+            remove_db(key_ptr);
+        }
+    }
+}
+
+#[cfg(feature = "iterator")]
+/// ExternalIterator makes a call out to next
+/// We only allow one open iterator at a time, so no need to pass references
+/// it automatically refers to result of last range call
+struct ExternalIterator {}
+
+#[cfg(feature = "iterator")]
+impl Iterator for ExternalIterator {
+    type Item = KV;
+
+    fn next(&mut self) -> Option<Self::Item> {
+        let key = alloc(MAX_READ);
+        let value = alloc(MAX_READ);
```

Comment:
Can you rename those to `key_ptr`/`value_ptr` to differentiate from the variable names below?